lib/src/request.dart

```diff
@@ -113,10 +113,10 @@ class Request extends BaseRequest {
           'content-type "application/x-www-form-urlencoded".');
     }
 
-    return Uri.splitQueryString(body, encoding: encoding);
+    return splitQueryString(body, encoding: encoding);
```

Comment:
Agree with your assessment, want evolved behavior rather than breaking behavior.  Multi-map would solve problems in other areas as well.  I'll need to bring myself up to speed on lower level details of Dart language and see if I can tackle this.  I do want to try, and will let you know if I'm up to the challenge.  Thanks